manim/scene/scene_file_writer.py

```diff
@@ -68,26 +68,18 @@ def init_output_directories(self):
         """
         module_directory = self.output_directory or self.get_default_module_directory()
         scene_name = self.file_name or self.get_default_scene_name()
-        #print("1")
-        #print(dirs.MEDIA_DIR)
-        if self.save_last_frame or self.save_pngs:
-            if dirs.MEDIA_DIR != "":
-                image_dir = guarantee_existence(os.path.join(
-                    dirs.MEDIA_DIR,
-                    "images",
-                    module_directory,
-                ))
+        if self.save_last_frame:
```

Comment:
We still need to check for `self.save_pngs` here, no?

Reply:
oh, you are right!